### DISCUSSION TITLE
mi práctica HTML

### DISCUSSION THREAD

**Discussion by MarcosSilvaG** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#287] [Show and tell]
Hola compañeros espero ya estén entregando aunque aún tenemos un poc de tiempo :D.
Aquí les muestro mi página [https://doux-complicite.000webhostapp.com/](url) espero les agrade. un saludo y ánimos !

**Comment by OscarPedroza** [comment]
Buenas noches, te quedo muy bien compañero, buena paleta de colores.

**Reply by MarcosSilvaG** [reply]
Acabo de revisarla y muestra boostrap xd yo ni sé usar esa cosa aún. Gracias para la paleta fui a la web que nos mostraron en el live con la persona que se dedicaba a UX/UI  y ahí estuve buscando hasta que encontré una que me llamó la atención.

**Reply by hackit92** [reply]
> Acabo de revisarla y muestra boostrap xd yo ni sé usar esa cosa aún. Gracias para la paleta fui a la web que nos mostraron en el live con la persona que se dedicaba a UX/UI y ahí estuve buscando hasta que encontré una que me llamó la atención.

En cuanto te metas un poco más te va a encantar usarlo, todo muy bien con tu página, solo no puedo ver el dashboard del pastelero :/

**Reply by MarcosSilvaG** [reply]
lo dejé en /pastelero.html con la idea de que solo el pastelero pudiera ingresar a ella y no todos los visitantes xd aunque es un poco "rudimentario" ya que haciendo un scaneo a la web podrían llegar igual pero esa era la idea inicial. :D

**Comment by JuanSalvador73** [comment]
Muy bien sitio, felicidades. :D

**Reply by MarcosSilvaG** [reply]
Gracias!!
